### ISSUE TITLE
Synchronicity Errors

### ISSUE THREAD

**Issue by Servinjesus1** [oleg-shilo/codemap.vscode#100]
Sorry for the new bug, not sure if it was introduced from the last patch (#96), but every time I switch files, I get a `stack trace: Error: Cannot resolve tree item for element` error referencing the previous document's structure.

This happens regardless of if default or custom configurations are used. However it may also be because something is super corrupt. I had JSON configurations in my settings.json, but after I removed it I could still run Edit mapper for current file and it would take me to the same line that the settings were at.

**Comment by Servinjesus1** [comment]
I tried uninstalling and reinstalling and inspecting the `User` directory. Will try restarting to see if something in a temp cache is bugged.

Otherwise this doesn't affect plugin usability but it is annoying to deal with lots of error messages when debugging other plugins, etc.

**Comment by oleg-shilo** [comment]
>  it is annoying to deal with lots of error messages

Of course. Can you help me to reproduce it so with the debugger I can pin point the routine that raises it and may be put a condition around it
